configure.ac

```diff
@@ -105,9 +105,8 @@ PKG_CHECK_MODULES(PCMK, [pacemaker],,
 		  [PKG_CHECK_MODULES(PCMK, [pcmk])])
 
 # Python casing, prefer 3.3+ to 2.{6...}
-AM_PATH_PYTHON([3.3], , [PYTHON=:])
-if test "x$PYTHON" = x:; then
-	AM_PATH_PYTHON([2.6])
```

Comment:
Haven't thought of that `PYTHON` environment variable implication
back in 31133e8, thanks for spotting and fixing.